tensorflow-core/tensorflow-core-api/build.sh

```diff
@@ -2,6 +2,10 @@
 # Script to build native TensorFlow libraries
 set -eu
 
+if [ ! -f .tf_configure.bazelrc ]; then
+    echo "WARNING: No .tf_configure.bazelrc, using CI settings.  For local builds, run the ./configure script." >&2
```

Comment:
I would remove the "WARNING:" prefix, it is legitimate and common to use the CI settings for a local build (e.g. on a different Linux distribution).